pom.xml

```diff
@@ -128,44 +128,30 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.6.1</version>
+        <version>3.8.0</version>
         <configuration>
-          <compilerId>javac-with-errorprone</compilerId>
-          <forceJavacCompilerUse>true</forceJavacCompilerUse>
-          <!-- maven-compiler-plugin defaults to targeting Java 5, but our javac
-          only supports >=6 -->
-          <source>1.8</source>
-          <target>1.8</target>
+          <!-- Use <release> instead when migrating to JDK 11. -->
```

Comment:
does release work with java8 as well? or do you have to use source/target for java8?

Reply:
`--release` flag is unknown to Java 8 compilers, so unfortunately, we will have to retain `--source` and `--target` for a very long time.

Reply:
Oh I didn't realize it was a pass-through directly to compiler.